README.md

```diff
@@ -47,7 +47,14 @@ Getting Started
     bin/ycsb run basic -P workloads/workloada
     ```
 
-  Running the `ycsb` command without any argument will print the usage. 
+   Running the `ycsb` command without any argument will print the usage. 
+   The 'multiGets' option is an optimization where a set of 'gets' can be batched (on the client side) and sent to the server in one
```

Comment:
sorry, I just noticed this is in the top-level README. it should be in the hbase-specific README, since it doesn't appear to impact any of the other datastore bindings.